azure-pipelines-official.yml

```diff
@@ -171,3 +175,10 @@ jobs:
   - task: ms-vseng.MicroBuildTasks.521a94ea-9e68-468a-8167-6dcf361ea776.MicroBuildCleanup@1
     displayName: Perform Cleanup Tasks
     condition: succeededOrFailed()
+
```

Comment:
You probably also need to add the steps to do this, https://github.com/dotnet/arcade/blob/master/eng/common/templates/job/job.yml#L191-L205, so that the "publish-build-assets" job can consume the manifests